core/src/main/kotlin/di/Module.kt

```diff
@@ -67,7 +67,6 @@ import org.eclipse.apoapsis.ortserver.services.DefaultAuthorizationService
 import org.eclipse.apoapsis.ortserver.services.InfrastructureServiceService
```

Comment:
Commit message: Please also mention that this is a fixup for the commit you created in your previous PR.